Formula/gitlab-ci-multi-runner.rb

```diff
@@ -25,39 +24,40 @@ class GitlabCiMultiRunner < Formula
   end
 
   resource "prebuilt-x86_64.tar.gz" do
-    url "https://gitlab-ci-multi-runner-downloads.s3.amazonaws.com/v1.3.3/docker/prebuilt-x86_64.tar.gz",
+    url "https://gitlab-ci-multi-runner-downloads.s3.amazonaws.com/v1.4.0/docker/prebuilt-x86_64.tar.gz",
       :using => :nounzip
-    version "1.3.3"
-    sha256 "5ae536cb7598607f1a0d4c0cb7f31d83d6c19e386106ea50192933a726f33ca9"
+    version "1.4.0"
+    sha256 "01246d4b97ff91e3ce7de258bf0987bee265ed71d647e5dbf10b104b7073a645"
   end
 
   def install
-    mkdir_p buildpath/"src/gitlab.com/gitlab-org"
-    ln_sf buildpath, buildpath/"src/gitlab.com/gitlab-org/gitlab-ci-multi-runner"
-
     ENV["GOPATH"] = buildpath
-
+    proj = "gitlab.com/gitlab-org/gitlab-ci-multi-runner"
+    dir = buildpath/"src/#{proj}"
+    dir.install buildpath.children
     ENV.prepend_create_path "PATH", buildpath/"bin"
     Language::Go.stage_deps resources, buildpath/"src"
 
     cd buildpath/"src/github.com/jteeuwen/go-bindata/go-bindata" do
       system "go", "install"
     end
 
-    resource("prebuilt-x86_64.tar.gz").stage do
-      system "go-bindata", "-pkg", "docker", "-nocompress", "-nomemcopy", "-nometadata", "-o", buildpath/"executors/docker/bindata.go", "prebuilt-x86_64.tar.gz"
-    end
-
-    cd "src/gitlab.com/gitlab-org/gitlab-ci-multi-runner" do
-      commit_sha = `git rev-parse --short HEAD`
-
-      # Disable vendor support for go 1.5 and above
-      ENV["GO15VENDOREXPERIMENT"] = "0"
+    cd dir do
+      resource("prebuilt-x86_64.tar.gz").stage do
+        system "go-bindata", "-pkg", "docker",
+               "-nocompress", "-nomemcopy", "-nometadata",
+               "-o", dir/"executors/docker/bindata.go",
+               "prebuilt-x86_64.tar.gz"
+      end
 
-      # Copy from Makefile
-      system "godep", "go", "build", "-o", "gitlab-ci-multi-runner", "-ldflags", "-X main.NAME=gitlab-ci-multi-runner -X main.VERSION=#{version} -X main.REVISION=#{commit_sha}"
+      commit_sha = Utils.popen_read("git", "rev-parse", "--short", "HEAD")
+      system "go", "build", "--ldflags="\
```

Comment:
For this at least, I'm inclined to copy `doctl.rb`, since it's a `go` formula, it's close to rebellion option 1, and it's slightly less repetitive than that since it factors out the `base_flag`.